todo_app/mongo.py

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
We don't want to keep the existing Trello behaviour so rather than creating this separate file we should override the `app.py` file - this becomes particularly important when making future changes because otherwise we have to update both - e.g. for M10 you've got the Trello version of the app authenticating but not the Mongo version!